### PR TITLE
updates rummage to v1.2

### LINKED ISSUE #189: Pagination UI and functionality
As a user

I want to be able to see 20 issues at a time, with the ability to go from page to page

so that there is not too much information on the screen at one time.

- [x] Calculate number of pages based on number of issues (25m)
- [x] Display sed number of page numbers (1h)
- [x] Build functionality so the page number corresponds to correct results (2h)
- [x] UI for pagination (25m)
- [x] See which page I am currently on (1h)
- [x] Functioning 'next' 'prev' buttons as well as pages (1h)
- [x] Ellipses for when there are more than 5 pages (1h)
screen shot 2017-07-29 at 15 02 37

### PR DESCRIPTION
**Pull request by finnhodgkin**
So only a limited number of pagination links display for large collections of issues:
woop

Also updates the issues per page count to 20

 fixes #189

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -33,6 +33,12 @@ config :ueberauth, Ueberauth.Strategy.Github.OAuth,
   client_id: System.get_env("GITHUB_CLIENT_ID"),
   client_secret: System.get_env("GITHUB_CLIENT_SECRET")
 
+config :rummage_ecto, Rummage.Ecto,
+  default_repo: Tudo.Repo,
+  default_per_page: 20,
```
Comment (Jbarget) [inline config/config.exs]: do we need both `default_per_page ` and `max_page`?

**Review by Jbarget — COMMENTED** [review]
good work!

**Comment by shouston3** [comment]
Nice guys, pagination looks and works great

**Review by samhstn — APPROVED** [review]
Nice guys, pagination looks and works great

File: config/config.exs (hunk 1 above)
Comment (finnhodgkin) [reply config/config.exs]: No sorry max_page is now doing nothing. I'll fix it.

**Review by Jbarget — APPROVED** [review]

**Pull request merged**